packages/vue3/source/mobile/src/init.ts

```
@@ -0,0 +1,205 @@
+
+import App from "./App.vue";
+import { setConfig } from "./setConfig";
+
+import "./index.css";
+
+configureCompat({
+  MODE: 3,
+})
+
+const evalWrap = function (metaData, fnName) {
+  metaData && fnName && metaData?.frontendEvents[fnName] && eval(metaData.frontendEvents[fnName]);
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)